### ISSUE TITLE
FileNotFoundError: [Errno 2] 没有那个文件或目录: 'screen'

### ISSUE THREAD

**Issue by ImFyatto** [ranger/ranger#2070]
#### Runtime Environment

- Operating system and version: Ubuntu 20.04, kernel 5.4.0-42
- Terminal emulator and version: terminator 1.91, but running tmux 3.0a
- Python version: 3.8.2
- Ranger version/commit: 1.9.3
- Locale: zn_CH.UTF-8

#### Current Behavior
super user running(sudo)

#### Steps to reproduce
sudo ranger

#### Traceback

```
ranger version: ranger 1.9.3
Python version: 3.8.2 (default, Jul 16 2020, 14:00:26) [GCC 9.3.0]
Locale: zh_CN.UTF-8

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/ranger/core/main.py", line 171, in main
    fm.initialize()
  File "/usr/lib/python3/dist-packages/ranger/core/fm.py", line 132, in initialize
    self.ui.initialize()
  File "/usr/lib/python3/dist-packages/ranger/gui/ui.py", line 127, in initialize
    self.handle_multiplexer()
  File "/usr/lib/python3/dist-packages/ranger/gui/ui.py", line 499, in handle_multiplexer
    self._screen_title = check_output(
  File "/usr/lib/python3/dist-packages/ranger/ext/spawn.py", line 35, in check_output
    process = Popen(popenargs, stderr=fd_devnull, **kwargs)
  File "/usr/lib/python3.8/subprocess.py", line 854, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.8/subprocess.py", line 1702, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] 没有那个文件或目录: 'screen'

ranger crashed. Please report this traceback at:
https://github.com/ranger/ranger/issues

```
### Other
Ordinary user and root user can normal operation,but... (>_<)
**try edit /etc/sudoers and add:**
```
Defaults    !env_reset
```
**???????**

**Comment by toonn** [comment]
Can you check whether this was fixed on master?

**Comment by ImFyatto** [comment]
sure

**Comment by ImFyatto** [comment]
> Can you check whether this was fixed on master?

sure

**Comment by ImFyatto** [comment]
@toonn  sure

**Comment by ImFyatto** [comment]
@toonn The master branch is OK!

**Comment by toonn** [comment]
Fixed in #1852.
